README.rst

```diff
@@ -628,8 +628,8 @@ Allows to specify backend URL for API requests, if the frontend is hosted separa
 Allows to specify a `list` of tuples for adding commands as described in
 `'How to add commands" <#how-to-add-commands>`_ section.
 
-``OPENWISP_CONTROLLER_DEVICE_GROUP_SCHEMA``
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Hey, why this change? It was better before!

Reply:
Makes it consistent with `VPNCLIENT` and explicitly show that DeviceGroup is it's own separate thing.